src/EA.Iws.Web/Areas/AdminImportNotificationMovements/Views/Capture/Create.cshtml

```diff
@@ -51,4 +84,10 @@
             $("#ShipmentNumber").focus();
         });
     </script>
+<script src="~/Scripts/jquery.stickem.js"></script>
```

Comment:
As above, you can move the script to the bundle and move the `stickem()` call to the existing initializer.